server/settings/_prodbase.py

```diff
@@ -21,18 +22,17 @@ class Config(object):
 
     RAVEN_IGNORE_EXCEPTIONS = raven_exceptions
 
-    # Service Keys
-    GOOGLE = {
-        'consumer_key': os.environ.get('GOOGLE_ID'),
-        'consumer_secret':  os.environ.get('GOOGLE_SECRET')
-    }
-
```

Comment:
the sendgrid section is also in __init__.py

Reply:
Also, `TENAND_ID`can be the domain name of the Active Directory. So either a custom domain or the assigned `foobar.onmicrosoft.com` one

Reply:
^ updated comment